src/network/clientpackethandler.cpp

```diff
@@ -1815,4 +1815,12 @@ void Client::handleCommand_SetLighting(NetworkPacket *pkt)
 	}
 	if (pkt->getRemainingBytes() >= 4)
 		*pkt >> lighting.volumetric_light_strength;
+	if (pkt->getRemainingBytes() >= 24) {
```

Comment:
I think it is worth to change the check for the bytes availability in the packet doing it similar with https://github.com/minetest/minetest/pull/14343/files#diff-592338f1bfb0275512da218625ce67e993b91984c175f70fe376825ab4e0aad2R1804

Reply:
This is the normally used style.
I will change it in the rebase after PR #14343 is merged.